blockly-rtc/src/WorkspaceClient.js

```diff
@@ -155,60 +153,69 @@ export default class WorkspaceClient {
       if (rows.length == 0) {
         return eventQueue;
       };
-  
+
       this.lastSync = rows[rows.length - 1].serverId;
   
       // No local changes.
       if (this.notSent.length == 0 && this.inProgress.length == 0) {
         rows.forEach((row) => {
-          eventQueue.push(this.createWorkspaceAction_(row.event, true));
+          eventQueue.push.apply(
```

Comment:
Why do you need to use `push.apply` instead of `push` here?

Reply:
`this.createWorkspaceAction` would return one object at a time, so I added each object to the `eventQueue` using `push`.
Now, `this.createWorkspaceActions` returns an array of objects, so I am using `push.apply` to add the objects to `eventQueue` individually rather than using `push` which would add an array.